variable.c

```diff
@@ -2672,6 +2724,8 @@ rb_autoload_load(VALUE module, ID name)
 
     // Only one thread will enter here at a time:
     return rb_mutex_synchronize(arguments.mutex, autoload_try_load, (VALUE)&arguments);
+
+    // rb_mutex_synchronize(autoload_mutex, autoload_feature_clear_if_empty, (VALUE)&arguments.autoload_data);
```

Comment:
Do you need this comment? I think we currently get an unused function warning because `autoload_feature_clear_if_empty` is not used here.

Reply:
I'll remove it in another PR coming in the next day.